index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Victors Profile</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+</head>
+
+<body>
+   
+    <header>
+        <nav>
+            <a href="#somelink" id="home-link">Home</a> 
+        </nav> 
+        <img src="https://assured-dp.com/wp-content/uploads/2017/12/male-placeholder.jpg" alt="Victor">
+        <h1>Victor Ren</h1>
+    </header>
+
+
+<main>
+    <section class="row">
+        <div class="column square" id="t-left"> 
+        <h3>About</h3>
+         <ul>
+            <li>Favorite Food: Pizza</li>
+            <li>Hobbies: Arts & Crafts</li>
+            <li>Something else: another fact</li>
+        </ul>
+        </div>
+        <img class="column square" id="about-img" src="https://mskassist.com/wp-content/uploads/2020/03/placeholder.png" alt="inline image">       
```

Comment:
Alt tags should be treated like a caption which explains the context the image gives to the webpage. If the image was broken, it would be in its place. We know from the 'img' tag that there should be an image there, so avoid adding that in your alt description. If it was a photograph of Victor at his desk, you would put alt as 'Victor sitting at his desk' instead of 'image of victor'.